tests/Microsoft.Identity.Test.Unit/AppConfigTests/PublicClientApplicationBuilderTests.cs

```diff
@@ -640,19 +641,50 @@ public void IsBrokerAvailable_net5()
         public void IsBrokerAvailable_OldDotNet()
         {
             var builder1 = PublicClientApplicationBuilder
-                    .Create(TestConstants.ClientId);
+                    .Create(TestConstants.ClientId)
+                    .WithAuthority(TestConstants.AuthorityTenant);
 
             // broker is not available out of the box
             Assert.AreEqual(false, builder1.IsBrokerAvailable());
 
             var builder2 = PublicClientApplicationBuilder
                    .Create(TestConstants.ClientId)
-                   .WithDesktopFeatures();
+                   .WithDesktopFeatures()
+                   .WithAuthority(TestConstants.AuthorityTenant);
 
 
             // broker is not available out of the box
             Assert.AreEqual(DesktopOsHelper.IsWin10OrServerEquivalent(), builder2.IsBrokerAvailable());
         }
+
+        [TestMethod]
+        public void NoBrokerADFS_OldDotNet()
```

Comment:
This test will run on: .net classic, .net core and .net5 (which is probably what you want).